docs/source/user_guide/calculations.rst

```diff
@@ -32,4 +34,36 @@ They will be converted to AiiDA data types by the script itself.
 
 .. code-block:: python
 
-    verdi run submit_singlepoint.py janus@localhost --calctype "singlepoint" --structure "path/to/structure" --model "path/to/model" --precision "float64" --device "cpu"
+    verdi run submit_singlepoint.py janus@localhost  --structure "path/to/structure" --model "path/to/model" --precision "float64" --device "cpu"
+
+
+Geometry Optimisation calculation
+---------------------------------
+
+A `GeomOpt` Calculation represents a `Calcjob` object within the AiiDA framework.
+
+
+Usage
+^^^^^
+
+This calculation can be executed using either the `run` or `submit` AiiDA commands.
+Below is a usage example with some additional geometry optimisation parameters. These parameters must be AiiDA data types.
+
+
+.. code-block:: python
+
+
+    GeomOptCalculation = CalculationFactory("janus.opt")
+    submit(GeomOptCalculation, code=InstalledCode, structure=StructureData, max_force=Float(0.1), vectors_only=Bool(False), fully_opt=Bool(True))
```

Comment:
Just to note, once janus is updated, you shouldn't need `--fully-opt` as well as `--vectors-only` (although it should work either way)

Reply:
ok, this is just an example, but so how would it work then? if I put vectors-only True it's gomnna be fully-opt True automatically?

Reply:
Effectively, yes.

Previously you needed to either have both, or just `--fully-opt`, but it doesn't really make sense to require both when it's clear enough just from `--vectors-only` what you want, and on the Python side it doesn't make sense to set `hydostatic_strain=True` without a filter.

You could even argue for a more extreme change where you have to pick one or the other, since with `--vectors-only` you're not actually fully optimizing, but I think it's probably fine.

Reply:
I'm open to changing it again, if an alternative is more intuitive.